api/app/models/user.rb

```diff
@@ -2,26 +2,28 @@
 #
 # Table name: users
 #
-#  id                :bigint(8)        not null, primary key
-#  academic_year     :integer(4)
-#  catalog_year      :integer(4)
-#  coop_cycle        :string
-#  courses_completed :json             default([]), is an Array
-#  courses_transfer  :json             default([]), is an Array
-#  email             :string           default(""), not null
-#  graduation_year   :integer(4)
-#  image_url         :string
-#  is_advisor        :boolean          default(FALSE), not null
-#  major             :string
-#  username          :string
-#  created_at        :datetime         not null
-#  updated_at        :datetime         not null
-#  nu_id             :string
+#  id                     :bigint(8)        not null, primary key
+#  academic_year          :integer(4)
+#  coop_cycle             :string
+#  email                  :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  graduation_year        :integer(4)
+#  image_url              :string
+#  is_advisor             :boolean          default(FALSE), not null
+#  major                  :string
+#  remember_created_at    :datetime
+#  reset_password_sent_at :datetime
+#  reset_password_token   :string
+#  username               :string
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#  nu_id                  :string
```

Comment:
These schema annotations reflect what our migrations build, so these should be in master. If these comments change when you don't expect them to, try doing a rails db:drop db:create db:migrate to your database.